conf/app.routes

```diff
@@ -18,7 +18,9 @@ GET           /individuals/subcontractor/idms/taxpayer/:idKey
 
 
 POST          /individuals/debts/time-to-pay/quote                                                          uk.gov.hmrc.debttransformationstub.controllers.TimeToPayController.generateQuote
-POST          /debts/time-to-pay/quote                                                          uk.gov.hmrc.debttransformationstub.controllers.TimeToPayController.generateQuote
+POST          /debts/time-to-pay/quote                                                                      uk.gov.hmrc.debttransformationstub.controllers.TimeToPayController.generateQuote
+
+POST          /debts/time-to-pay/affordability/affordable-quotes                                            uk.gov.hmrc.debttransformationstub.controllers.TimeToPayController.generateAffordabilityQuote
```

Comment:
I just thought sorry, the stub has that annoying feature switch turned on. You'll need
`/individuals/time-to-pay/affordability/affordable-quotes` as well